Hakawai/Mentions/HKWMentionsStartDetectionStateMachine.m

```diff
@@ -232,13 +236,14 @@ - (void)deleteTypedCharacter:(unichar)deletedChar withCharacterNowPrecedingCurso
                 self.charactersSinceLastWhitespace = 0;
                 self.stringBuffer = [@"" mutableCopy];
             }
+            break;
         }
         case HKWMentionsStartDetectionStateQuiescentStalled: {
             // Change state to QuiescentReady when either:
             //   1. A whitespace character is encountered
             //   2. A NON-whitespace character is encountered and a whitespace character was deleted
             //   3. A punctuation character is encountered
-            if (currentCharacterType == CharacterTypeSeparator
```

Comment:
why did this change, from current to previous, both here and above

Reply:
As per your previous review comment(https://github.com/aniyati/Hakawai/pull/1#discussion_r347678752) and as the character is before the deleted character, we have renamed `currentCharacterType` as `precedingCharacterType`.